packages/account/src/Styles/account.scss

```diff
@@ -1182,3 +1182,207 @@ $MIN_HEIGHT_FLOATING: calc(
 .proof-of-identity {
     height: 100%;
 }
+
+/** @define deactivate-account; weak */
+.deactivate-account {
+    max-width: 67.2rem;
+
+    .confirm-all-close {
+        &__information {
+            color: var(--text-prominent);
+            font-size: var(--text-size-xs);
+            line-height: 1.43;
+            margin-bottom: 8px;
+
+            &--bold {
+                font-weight: bold;
+                margin-bottom: 8px;
+                margin-top: 5px;
+            }
+        }
+        &__steps {
+            margin-top: 16px;
+            color: var(--text-prominent);
+            font-size: var(--text-size-xs);
+            line-height: 1.43;
+
+            &__title {
+                font-weight: bold;
+            }
+            &__content {
+                color: var(--text-general);
+
+                &__link {
+                    color: var(--text-loss-danger);
+                    font-weight: bold;
+                    text-decoration: none;
+                }
+            }
+        }
+        &__button {
+            margin-top: 24px;
+            float: right;
+        }
+    }
+    .deactivate-account-reasons {
+        color: var(--text-prominent);
+        font-size: var(--text-size-xs);
+        line-height: 1.43;
+        font-weight: bold;
+
+        &__title {
+            margin-bottom: 18px;
+        }
+        &__checkbox {
+            margin-top: 10px;
+            width: fit-content;
+
+            .dc-checkbox__label {
+                font-weight: normal;
+            }
+        }
+        &__input {
+            margin: 16px 0 0;
+            max-width: 400px;
+            text-overflow: ellipsis;
+
+            > input {
+                min-height: 96px;
+            }
+        }
+        &__error {
+            color: var(--purchase-main-2);
+        }
+    }
+    @include mobile {
+        width: 92%;
+        margin: 0 auto;
+    }
+}
+
+/* stylelint-disable */
+.dc-modal__container_deactivate-account-reasons {
+    .dc-modal-header {
+        border: none;
+        min-width: 430px;
+    }
+    .account-closure-warning-modal {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        padding: 32px;
```

Comment:
can we use relative measurement `rem` instead 🙏

Reply:
sure